run_all_tests_unix.sh

```diff
@@ -8,13 +8,7 @@ test_files="$(grep ^[^#] test_classes_list.txt)"
 #    Change the test command and parameters accordingly.
 tty -s && tput bold
 
-# 2.1 Check for python interpreter version
-if [ -x "$(command -v python2)" ]; then
-    interpreter="python2"
-else
-    echo "No python2 command found! Will use the default python command."
-    interpreter="python"
-fi
+interpreter="python"
```

Comment:
We can't change this right now, our PR testing suite on Jenkins depends on this being Python 2.